repos/system_upgrade/common/actors/cephvolumescan/libraries/cephvolumescan.py

```diff
@@ -0,0 +1,55 @@
+import json
+import os
+import re
+
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import run
+from leapp.models import InstalledRPM
+
+CEPH_CONF = "/etc/ceph/ceph.conf"
+CONTAINER = "ceph-osd"
+
+
+def select_osd_container(engine):
+    container_name = ""
+    output = run([engine, 'ps'])
+    for line in output['stdout'].splitlines():
+        container_name = line.split()[-1]
+        if re.match(CONTAINER, container_name):
+            return container_name
+    return container_name
+
+
+def container_or_rpm():
+    container_binary = ''
+    base_cmd = ['ceph-volume', 'lvm', 'list', '--format', 'json']
+    if has_package(InstalledRPM, "ceph-common"):
+        return base_cmd
+    if has_package(InstalledRPM, "podman"):
+        container_binary = 'podman'
+    else:
+        container_binary = 'docker'
+    container_name = select_osd_container(container_binary)
+    container_cmd = [container_binary, 'exec', container_name]
+    container_cmd.extend(base_cmd)
+
+    return container_cmd
+
+
+def run_ceph_lvm_list():
+    command = container_or_rpm()
+    output = run(command)
```

Comment:
@asm0deuz are you sure the command will never end with non-zero exit code? if not, it should be handled nicely so customers cannot see just traceback. (`except CalledProcessError`). See the `StopActorExecutionError` exception to raise a nice error for leapp in such a case.